### ISSUE TITLE
Emojis are not removed

### ISSUE THREAD

**Issue by WoaDmulL** [davidmogar/cucco#47]
The new emojis like 🤗,  🥂,  🤔, 🤘fail to be removed.

Check this gist https://gist.github.com/octohedron/3823d081eb1b92abe93b570875ec77f4

**Comment by davidmogar** [comment]
Thank you for the report. I'll try to fix this as soon as possible.

**Comment by davidmogar** [comment]
I'm currently considering removing the current regex and use instead [this library](https://github.com/carpedm20/emoji) as it would allow to replace the emojis with text. That was the initial goal. What do you think?

**Comment by WoaDmulL** [comment]
I think it would be better to remove them by default; and replace them with some custom setting or optional argument passed to the function when needed. Or the opposite way.

**Comment by davidmogar** [comment]
Yeah, that is the current behavior. The problem with it is that you lose information in the process. Contemplate the next scenario:
* Today I'm :smile:
* Today I'm :cry:

In both cases, applying the current code you'd get `Today I'm` what removes completely the useful information.

Just something to think about...

**Comment by WoaDmulL** [comment]
We could cover all cases:
```
normalize("Today I'm 😄") => Today I'm 😄
strip_emoji("Today I'm 😄") => Today I'm
describe_emoji("Today I'm 😄") => Today I'm happy
```
